package.json

```diff
@@ -45,64 +45,64 @@
         "test:clean": "yarn build:clean && yarn test:ci",
         "test:debug": "node --inspect node_modules/.bin/jest --runInBand --watch",
         "test:watch": "yarn test --watch",
-        "test:wdio": "yarn workspace @sa11y/wdio test && yarn workspace @sa11y/browser-lib test ",
-        "postinstall": "husky install && rm -rf .git/hooks && ln -s ../.husky .git/hooks"
+        "test:wdio": "yarn wdio run wdio.conf.js --suite wdio --suite integration --suite browserLib < /dev/null",
```

Comment:
* wdio is not installed globally hence we need yarn
* the dev null redirection is to prevent wdio from erroring when run as part of git hook https://github.com/webdriverio/webdriverio/issues/4957

Reply:
If the dependencies are set up properly you shouldn't need the `yarn` there. Similar to how you have `jest` above without the yarn. Is it only an issue within the git hook? Maybe that's what's making things weird. 

Either way, it's not a blocker to merge IMO but I would verify and remove it's unnecessary before merging.

Reply:
Ah 🤦 got it thanks for the catch, fixed it now along with others. Can you please take a look.

Wish there was a prettier/eslint plugin for pkg json that could catch these issues.

Was hoping that this would fix the wdio redirection error, but it didn't.
```
$ wdio run wdio.conf.js --suite wdio --suite integration --suite browserLib
2021-04-08T19:19:21.672Z WARN @wdio/config:ConfigParser: pattern __tests__/**/*.ts did not match any file

sa11y/node_modules/@wdio/cli/node_modules/@wdio/config/build/lib/ConfigParser.js:156
            throw new Error(`spec file(s) ${cliArgFileList.join(', ')} not found`);
                  ^
Error: spec file(s) refs/heads/feat_include_selector 8edc592b39b3a9f7ee91ac094e17f839db7b7c56 refs/heads/feat_include_selector 0091390a8677ce942b30f9d6448f05fa43359fae not found
    at ConfigParser.setFilePathToFilterOptions (sa11y/node_modules/@wdio/cli/node_modules/@wdio/config/build/lib/ConfigParser.js:156:19)
    at ConfigParser.merge (sa11y/node_modules/@wdio/cli/node_modules/@wdio/config/build/lib/ConfigParser.js:59:39)
    at new Launcher (sa11y/node_modules/@wdio/cli/build/launcher.js:28:27)
    at launch (sa11y/node_modules/@wdio/cli/build/commands/run.js:119:22)
    at Socket.<anonymous> (sa11y/node_modules/@wdio/cli/build/commands/run.js:114:9)
    at Socket.emit (events.js:327:22)
    at Socket.EventEmitter.emit (domain.js:467:12)
    at endReadableNT (internal/streams/readable.js:1327:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
husky - pre-push hook exited with code 1 (error)
error: failed to push some refs to 'github.com:mohanraj-r/sa11y.git'
```